pages/mentors.vue

```diff
@@ -131,7 +131,48 @@
               topics="Agile, Web, Mobile (Android), Chatbot"
               linkedin="ajahncharles"
             >
-              Winner LINEHACK. Co&#8209;founder Phits.Tech. Polyglot programmer.
+              ผู้ชนะ LINEHACK ผู้ร่วมก่อตั้ง Pits.Tech โปรแกรมเมอร์หลายภาษา
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/RungsunPromprasith.jpg')"
+              name="Rangsun Promprasith"
+              thai-name="รังสรรค์ พรมประสิทธิ์"
+              topics="Strategy"
+            >
+              บริษัท คิวคิว (ประเทศไทย) จำกัด และ บริษัท วาย เอ็ม เอ็ม วาย จำกัด.
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/workshop4-speaker1.jpg')"
+              name="Wanitcha Wankhawisun"
+              thai-name="วณิชชา วรรคาวิสันต์"
+              topics="Online marketing"
+            >
+              บริษัท พาณิชย์ดิจิทัล จำกัด
+            </mentor>
+          </div>
+          <div class="column is-half-tablet is-one-third-desktop is-one-quarter-widescreen">
+            <mentor
+              :image-src="require('~/assets/images/people/Anuwut.jpg')"
+              name="Anuwat jongyindee"
+              thai-name="อนุวัฒน์ จงยินดี"
+              topics="Financial account, Strategy, Good governance risk management"
```

Comment:
Change to `Finance, Strategy, Risk management`

Reply:
I changed.